src/content/docs/en/guides/styling.mdx

```diff
@@ -469,6 +471,77 @@ import "../styles/global.css";
 ---
 ```
 
+#### Legacy Tailwind 3 support
+
+To add (or keep) support for Tailwind 3, you will need to have both `tailwindcss@3` and the official Astro Tailwind integration `@astrojs/tailwind` installed. Installing these dependencies manually is only used for legacy Tailwind 3 compatibility, and is not required for Tailwind 4. You will also need a legacy Tailwind configuration:
+
+<Steps>
+1. Install Tailwind and the Astro Tailwind integration to your project dependencies using your preferred package manager:
+
+   <PackageManagerTabs>
+     <Fragment slot="npm">
+     ```shell
+     npm install @astrojs/tailwind@5.1.5
+     ```
+     </Fragment>
+     <Fragment slot="pnpm">
+     ```shell
+     pnpm add @astrojs/tailwind@5.1.5
+     ```
+     </Fragment>
+     <Fragment slot="yarn">
+     ```shell
+     yarn add @astrojs/tailwind@5.1.5
+     ```
+     </Fragment>
+   </PackageManagerTabs>
```

Comment:
``````suggestion
1. Install Tailwind and the Astro Tailwind integration to your project dependencies using your preferred package manager:

   <PackageManagerTabs>
     <Fragment slot="npm">
     ```shell
     npm install tailwindcss@3 @astrojs/tailwind
     ```
     </Fragment>
     <Fragment slot="pnpm">
     ```shell
     pnpm add tailwindcss@3 @astrojs/tailwind@5.1.5
     ```
     </Fragment>
     <Fragment slot="yarn">
     ```shell
     yarn add tailwindcss@3 @astrojs/tailwind@5.1.5
     ```
     </Fragment>
   </PackageManagerTabs>
``````